android/fastlane/Fastfile

```diff
@@ -25,12 +25,12 @@ platform :android do
   desc "Deploy a new staging version to Google Play"
   lane :deploy_staging do
     gradle(task: "clean assembleDevRelease")
-    upload_to_play_store
+    upload_to_play_store(track: internal)
   end
 
   desc "Deploy a new version to Google Play"
   lane :deploy_mainnet do
     gradle(task: "clean assembleMainRelease")
-    upload_to_play_store
+    upload_to_play_store(track: rollout)
```

Comment:
As I understand the `rollout` track is for releasing gradually, so maybe we'd need to add the `rollout` parameter as well (it has no default value apparently [1]).

[1]: https://docs.fastlane.tools/actions/supply/#parameters

Reply:
I think the documentation is wrong 🤔 it actually has a default of 10%